lib/improver/nbhood.py

```diff
@@ -0,0 +1,161 @@
+# -*- coding: utf-8 -*-
```

Comment:
Longer term sort of question.. Do we have an easy way of checking this is in all new files and is correct?